### PR TITLE
fix the error for make verify

### PR DESCRIPTION
**Pull request by huone1**
Signed-off-by: Thor-wl <13164644535@163.com>

### CODE REVIEW

File: hack/update-gencode.sh [hunk 1]
```diff
@@ -22,16 +22,19 @@ echo $(dirname "${BASH_SOURCE[0]}")
...
 
+chmod 755 vendor/k8s.io/code-generator/generate-groups.sh
+chmod 755 vendor/k8s.io/code-generator/generate-internal-groups.sh
+
 # generate the code with:
 # --output-base    because this script should also be able to run inside the vendor dir of
 #                  k8s.io/kubernetes. The output-base is needed for the generators to output into the vendor dir
 #                  instead of the $GOPATH directly. For normal projects this can be dropped.
-${SCRIPT_ROOT}/hack/generate-groups.sh "deepcopy,client,informer,lister" \
+vendor/k8s.io/code-generator/generate-groups.sh "deepcopy,client,informer,lister" \
```
Comment (shinytang6) [inline hack/update-gencode.sh:32]: it's better to use `bash` here and rm chmod above

Comment (huone1) [reply hack/update-gencode.sh]: It's better. done

**Review by shinytang6 — APPROVED** [review]
/lgtm

**Pull request merged**